### DISCUSSION TITLE
How do I change S3 bucket to a other one?

### DISCUSSION THREAD

**Discussion by SJang1** [mastodon/mastodon#19470] [Q&A]
I am using S3 Bucket on AWS rn, and running the server. and now I want to change the S3 bucket to other company like GCS. What should I do to change the bucket? I would like to purge S3 in AWS, and create whole new one on GCS.

**Comment by Gargron** [comment]
There are tools like aws-cli and s3cmd that would allow you to transfer files from one bucket to another. Transfer without impact on end-users is a bit more tricky, since the transfer will miss files uploaded or deleted during the transfer. For something like that you would need a professional service.
